scripts/get_stats.py

```diff
@@ -1,17 +1,29 @@
+from dataclasses import dataclass
 from glob import glob
 
+@dataclass
+class BenchmarkStats:
+    get: []
+    post: []
+    average_get: None
+    average_post: None
+    average_total: None
+    total_requests: int
+    total_failures: int
+    error_percentage: int
+
+    def __init__(self):
+        self.get = []
+        self.post = []
+        self.average_get = None
+        self.average_post = None
+        self.total_requests = 0
+        self.total_failures = 0
+        self.error_percentage = 0
```

Comment:
The `init` isn't needed, that is the idea of dataclasses. You can set default values above.